core/che-core-api-core/src/main/java/org/eclipse/che/api/core/util/StubLineConsumer.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2012-2018 Red Hat, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.che.api.core.util;
+
+/**
+ * Line consumer which does nothing.
+ *
+ * @author Dmytro Nochevnov
+ */
+public class StubLineConsumer implements LineConsumer {
```

Comment:
There is https://github.com/eclipse/che/blob/master/core/che-core-api-core/src/main/java/org/eclipse/che/api/core/util/AbstractLineConsumer.java

Reply:
I know, but AbstractLineConsumer requires implementation and here it is.

Reply:
https://github.com/eclipse/che/blob/2c2bd6637c8169018e2b33397f9d738bbf82a0b1/core/che-core-api-core/src/main/java/org/eclipse/che/api/core/util/LineConsumer.java#L26
:)